src/components/Reviews.jsx

```diff
@@ -2,21 +2,49 @@ import { fetchReviews } from "../api";
 import { useState, useEffect } from "react";
 import { useParams } from "react-router-dom";
 import ReviewCard from "./ReviewCard";
+import { FormControl, InputLabel, NativeSelect } from "@mui/material";
 
 const Reviews = () => {
   const { category } = useParams();
   const [reviews, setReviews] = useState();
+  const [sortBy, setSortBy] = useState("created_at");
+  const [order, setOrder] = useState("desc");
   const [isLoading, setIsLoading] = useState(true);
+  const handleSortBy = (e) => {
+    setSortBy(e.target.value);
+  };
+  const handleOrder = (e) => {
+    setOrder(e.target.value);
+  };
   useEffect(() => {
-    fetchReviews(category).then((reviews) => {
+    fetchReviews(category, sortBy, order).then((reviews) => {
       setReviews(reviews);
       setIsLoading(false);
     });
-  }, [category]);
+  }, [category, sortBy, order]);
   return isLoading ? (
     <p>Loading reviews...</p>
   ) : (
     <>
+      <FormControl id="sort-menu">
+        <InputLabel variant="standard" htmlFor="uncontrolled-native">
+          Sort by
+        </InputLabel>
+        <NativeSelect onChange={(e) => handleSortBy(e)}>
+          <option value="created_at">Date</option>
+          <option value="comment_count">Comments</option>
+          <option value="votes">Votes</option>
+        </NativeSelect>
+      </FormControl>
+      <FormControl id="order-menu">
+        <InputLabel variant="standard" htmlFor="uncontrolled-native">
+          Order
+        </InputLabel>
+        <NativeSelect onChange={(e) => handleOrder(e)}>
+          <option value="desc">↑</option>
+          <option value="asc">↓</option>
```

Comment:
I would interpret an up arrow as asc and a down arrow as desc, if you're just using symbols be aware that they can be open to interpretation.